### PR TITLE
GUI-1609 ensure we only pass DelegateAccount to API calls when it is set

### PR DESCRIPTION
**Pull request by dkavanagh**
ensure we only pass DelegateAccount to API calls when it is set

https://eucalyptus.atlassian.net/browse/GUI-1609

### CODE REVIEW

**Comment by dkavanagh** [comment]
pep8 cleanup coming in the morning.

**Comment by euca-qa-gui-tester** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com//job/user-console-PR-monitor/113/
Test FAILed.

**Comment by euca-qa-gui-tester** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com//job/user-console-PR-monitor/114/
Test FAILed.

**Comment by euca-qa-gui-tester** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com//job/user-console-PR-monitor/115/
Test PASSed.

File: eucaconsole/models/quotas.py [hunk 1]
```diff
@@ -90,124 +91,200 @@ def create_quota_policy(self, view, user=None, account=None, as_account=''):
...
+            )
+            #  elb
+            self._update_quota_limit_(
+                view, policy_list, new_stmts,
+                'elb_load_balancers_max', 'elasticloadbalancing:createloadbalancer',
+                'elasticloadbalancing:quota-loadbalancernumber'
+            )
 
             # save policies that were modified
             for i in range(0, len(policy_list)):
```
Comment (kamalgill) [inline eucaconsole/models/quotas.py]: @dkavanagh: Minor point (no need to fix it here): we could use `for idx, value in enumerate(policy_list):` here to clean up this code block.  See https://docs.python.org/2/library/functions.html#enumerate

File: eucaconsole/views/users.py [hunk 2]
```diff
@@ -362,13 +368,16 @@ def user_avail_groups_json(self):
     @view_config(route_name='user_policies_json', renderer='json', request_method='GET')
     def user_policies_json(self):
         """Return user policies list"""
-        as_account = self.request.params.get('as_account', '')
+        as_account = self.request.params.get('as_account', None)
```
Comment (kamalgill) [inline eucaconsole/views/users.py]: @dkavanagh: Minor point, Python's dict.get(key) defaults to None when the lookup fails, so no need to explicitly fall back to None

**Pull request merged**